addons/event/models/event_ticket.py

```diff
@@ -143,6 +158,20 @@ def _compute_display_name(self):
                 )
             ticket.display_name = name
 
+    def _get_current_limit_per_order(self, event_slot_id=False):
```

Comment:
Actually shouldn't we call ``_get_seats_availability`` (and replace ``None`` results which says "no limit"' by EVENT_MAX_TICKETS) ?